src/arm/BBORG_MOTOR-00A2.dts

```diff
@@ -1,5 +1,8 @@
 /*
- * Copyright (C) 2018 Robert Nelson <robertcnelson@gmail.com>
+ * Copyright (C) 2020 Deepak Khatri <deepaklorkhatri7@gmail.com>
```

Comment:
Right approach is to *add* your copyright, not remove someone else's.

Reply:
The new code is very different from the old code that's why I dropped that. I will do that if you say so, talked to ds2 he suggested to include the comment you are seeing.